android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
     <uses-permission android:name="android.permission.ACCESS_NOTIFICATION_POLICY"/>
     <uses-permission android:name="android.permission.VIBRATE"/>
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.BLUETOOTH"/>
```

Comment:
why BLUETOOTH?

Reply:
WebView sometimes shows a warning that BLUETOOTH Permission is not present in the Manifest, so I added it to prevent future issues